core/dom/element.js

```diff
@@ -2044,20 +2044,20 @@ CKEDITOR.dom.element.clearMarkers = function( database, element, removeFromDatab
 		},
 
 		/**
-		 * Returns first element within this element that matches specified `selector`.
+		 * Returns the first element within this element that matches specified `selector`.
```

Comment:
Returns the first element within this element that matches the specified `selector`.